packages/widget-message-meet/src/app.js

```diff
@@ -24,7 +25,7 @@ function loadAllWidgets() {
   }
 }
 
-loadAllWidgets();
+document.addEventListener(`DOMContentLoaded`, loadAllWidgets, false);
```

Comment:
Added this because when using the widget on codepen, it would attempt to load before it was ready.